React-Template-master/src/i18n/config.ts

```diff
@@ -1,14 +1,15 @@
 import i18n from 'i18next';
-import { initReactI18next } from 'react-i18next';
-import translation from './en/translation.json';
-
-export const resources = {
-    en: {
-        translation
-    }
-} as const;
+import { initReactI18next, useTranslation } from 'react-i18next';
+import en_translation from './en/translation.json';
 
 i18n.use(initReactI18next).init({
     lng: 'en',
-    resources
+    resources: {
+        en: { translation: en_translation }
+    }
 });
+
+// Function to change lanugage
+export const changeLanguage = (language: string): void => {
```

Comment:
@GoutamShanbhag What if,  the user gives a wrong language, since the type is string the user can enter any value. How are you going to handle this situation ?

Reply:
The user will be changing the language using the dropdown option so the user can't give any random values.